pages/business/company/[companyId]/checkins.tsx

```diff
@@ -29,45 +32,68 @@ const CheckinsPage: React.FC<WithOwnerProps> = () => {
       <Box height={2} />
       <ActionList grid>
         {company?.areas.map((area) => {
-          const tickets = ticketsByArea?.[area.id]
-          const openCount = tickets?.open.length ?? 0
-          const closedCount = tickets?.closed.length ?? 0
-
-          return (
-            <ActionCard
-              key={area.id}
-              href="/business/company/[companyId]/area/[areaId]"
-              as={`/business/company/${companyId}/area/${area.id}`}
-            >
-              <ActionCard.Main
-                title={area.name}
-                subtitle={
-                  <>
-                    <span css={{ whiteSpace: 'nowrap' }}>
-                      am Tisch: {openCount}
-                    </span>
-                    {' – '}
-                    <span css={{ whiteSpace: 'nowrap' }}>
-                      ausgecheckt: {closedCount}
-                    </span>
-                  </>
-                }
-              />
-              <ActionCard.Below>
-                <Row flexWrap="wrap" flex="1 0">
-                  {[...Array(openCount)].map((_v, i) => (
-                    <Box key={i} mx="2px">
-                      <Icon icon={TinyCheck} size={4} />
-                    </Box>
-                  ))}
-                </Row>
-              </ActionCard.Below>
-            </ActionCard>
-          )
+          return <CheckinsActionCard area={area} companyId={companyId} />
         })}
       </ActionList>
     </OwnerApp>
   )
 }
 
+const CheckinsActionCard: React.FC<{
```

Comment:
Could you move this component into the block-directory, please?

Reply:
Nagut